### ISSUE TITLE
Contour.area should call cvContourArea

### ISSUE THREAD

**Issue by jondo** [atduskgreg/opencv-processing#82]
... instead of just giving the area of the bounding box.

**Comment by yaxu** [comment]
Agreed. The documentation says that "In most cases, this is a good approximation for the Contour's area", but I disagree. If you for example want to find out how spiky a contour is, you'll want to find the actual area and compare it with the perimeter.
